script.js

```diff
@@ -1,5 +1,13 @@
 let projectData = [
   {
+    projectName: "Interest Calculator",
```

Comment:
add ProjectImage as well

Reply:
How to add ProjectImage ?

Reply:
Create a gif of your project save it in assets/gif and
add the link in script.js